tuplex/core/src/physical/TransformTask.cc

```diff
@@ -738,6 +739,7 @@ namespace tuplex {
                                            size_t rangeStart, size_t rangeSize,
```

Comment:
we should prob. add a doxygen to setinputfilesource in TransformTask.h and explain there what the new partitionSize param means.